sw/host/opentitantool/src/command/update_usr_access.rs

```diff
@@ -21,6 +22,12 @@ pub struct UpdateUsrAccess {
 
     #[structopt(name = "OUTPUT_FILE")]
     output: PathBuf,
+
+    #[structopt(long,
+                name = "USR_ACCESS_VALUE",
```

Comment:
The `name` parameter is not needed.  Structopt defaults to naming the parameters the same name as the struct field name.  The only time it should be used is to rename parameters or provide a name to a positional parameter.

Reply:
I added this so that `--help` prints
```
--usr-access <USR_ACCESS_VALUE>    New USR_ACCESS value, default = timestamp
```

instead of

```
--usr-access <usr-access>    New USR_ACCESS value, default = timestamp
```

Removed for consistency.